lib/licensee/project_files/license_file.rb

```diff
@@ -64,6 +64,8 @@ class LicenseFile < Licensee::ProjectFiles::ProjectFile
         ^(creative\ commons\ )?Attribution-(NonCommercial|NoDerivatives)
       /xi
 
+      SPDX_REGEX = /\A[a-z\-0-9. +()]+#{PREFERRED_EXT_REGEX}+/i
```

Comment:
I suspect this would fail to match some LicenseRef-[idstring] files because idstring need not start with a number https://reuse.software/spec-3.3/#license-files